style.css

```diff
@@ -0,0 +1,19 @@
+.references{
```

Comment:
hey harun it looks great but i think it is better to `.references{border: solid 3px black}`. we have to work depend on the wire-frame.

Reply:
And it needs margin to make space b/n the boxes.